platform/feature-api/api/src/pod-api/uri.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Creates a random UUID string with a random hexadecimal value for each character in the string
+ * 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx', and returns the result.
+ * @returns a string in UUID format
+ */
+export function createUUID(): string {
+    return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(
+        /[xy]/g,
+        function (c) {
+            const r = (Math.random() * 16) | 0,
+                v = c == "x" ? r : (r & 0x3) | 0x8;
+            return v.toString(16);
+        }
+    );
+}
+
+export function isPolypodUri(uri: string): boolean {
+    return /^polypod:\/\//.test(uri);
+}
+
+export class PolyUri {
+    public readonly Uri: string;
+
+    constructor() {
+        this.Uri = `polypod://${createUUID()}`;
```

Comment:
The format for a polypod URI is `polypod://FeatureFiles/{id}`. That's what the other platforms use.

Reply:
Not in BrowserPod... So we really have to sit down and agree on a certain format, that's the same for everyone

Reply:
Those are not using these functions to generate the URIs either. Or the FS formats are interchangeable. Even if we do, wouldn't it make much more sense to use the feature name as part of the URI?

Reply:
Yes, we need to agree on a format. The nice thing is, with the move to core, that we can have everything in one place.

Reply:
I am not sure if we should add the feature name to the uri. I don't see any pros, and also no cons.

Reply:
You should not be able to change the Feature name and somehow try to access the resources of another feature. So I would keep the feature name out of the polypod uri.

Reply:
At any rate, I would say that's outside the scope of this PR. I'm simply using the format that's already used by pod.js, which is independent of the one used by Swift and Kotlin. Don't mind changing it, though, if it makes it more consistent. After all, we're encapsulating the URI generation, we can change it back to whatever we decide or actually code in the new core.
Don't particularly like the `FeatureFile` thing, since we already know it's a file for a feature, and it assumes there would be some other files for some other what exactly? But for the sake of consistency, won't strongly oppose it either.

Reply:
I also don't see a reason to have `FeatureFiles` constraint, it is up to the feature where it wants to store the files.

Also, the PolyPod should make sure that it creates the proper sandbox for the feature. This means, that any `path` sent from the feature should be relative to the `root` for that specific feature created by the PolyPod. Then the feature can do whatever it wants/needs.

Reply:
@GhenadiePusca I'm glad that we've opened, so to speak, that sandbox :-) However, I guess we have a consensus that we're good with generating the URI this way, at least initially, right?
My intention for putting it here is to carry it over to MockPod in the PR I've already indicated in the body.